README.md

```diff
@@ -329,6 +330,25 @@ descriptors:
           unit: second
 ```
 
+#### Example 5
+
+We can also define unlimited rate limit descriptors:
+
+```yaml
+domain: internal
+descriptors:
+  - key: ldap
+    rate_limit:
+      unlimited: true
+
+  - key: azure
+    rate_limit:
+      unit: minute
+      requests_per_unit: 100
+```
+
+For an unlimited descriptor, the request will not be sent to the underlying cache (Redis/Memcached), but will be quickly returned locally by the ratelimit instance.
```

Comment:
Can you mention why someone would want to do this? I assume for stats, etc.?

Reply:
Well, stats yes. But also, in our case, our client will not allow a request to go through if a descriptor is not found, so we do have to have it defined.

Reply:
I'll add something to the readme

Reply:
Added